js/utils.go

```diff
@@ -110,5 +110,5 @@ func contextIdentityArg(value js.Value) (immutable.Option[acpIdentity.Identity],
 	if err != nil {
 		return immutable.None[acpIdentity.Identity](), err
 	}
-	return immutable.Some(identity), nil
+	return immutable.Some(acpIdentity.Identity(identity)), nil
```

Comment:
nitpick: In other locations you used this patter instead: `immutable.Some[acpIdentity.Identity](identity)`. For consistancy it would be nice to use the same here.